### ISSUE TITLE
Add strings

### ISSUE THREAD

**Issue by danmichaelo** [scriptotek/alma-slipsomat#3]
Need to add support for pulling/pushing the translatable strings as well

**Comment by th122** [comment]
That would be very much appreciated!
So far the only "Backup" procedure seems to be either field-by-field _cut-and-paste_ to some text or spreadsheet document or something sophisticated, such as _"just take a screenshot whenever you change it"_. Neither is by any means convenient...

**Comment by danmichaelo** [comment]
Agree. Interestingly, there is actually the [Retrieve Code-table API](https://developers.exlibrisgroup.com/alma/apis/conf/GET/gwPcGly021p29HpB7XTI4K7cQ0vuYHLS4NSgDGmcRpRYqx5hIMRTng9SIKO5Vof+/37088dc9-c685-4641-bc7f-60b5ca7cabed) that we could get read the values from. Unfortunately, there is no write API.. I asked some time ago if it was in the planning, but it was not so. So updating the values would have to be done using the same Selenium approach, but I'm afraid it will be somewhat complex to make it work well.
